config/blockchain.js

```diff
@@ -72,9 +85,13 @@ module.exports = {
     syncMode: "light",
     rpcCorsDomain: "http://localhost:8000",
     wsOrigins: "http://localhost:8000",
-    account: {
-      password: "config/livenet/password"
-    }
+    accounts: [
+      {
+        nodeAccounts: {
+          password: "config/development/password" // Password to unlock the account
```

Comment:
```suggestion
          password: "config/livenet/password" // Password to unlock the account
```

Reply:
Thanks I noticed that as well, updated all of them (`privatenet, testnet, livenet`). Updated now.